app/services/ipam/db.rb

```diff
@@ -4,8 +4,9 @@ class Db < Base
     def suggest_ip
       subnet_range.each do |address|
         ip = address.to_s
-        if !excluded_ips.include?(ip) && !subnet.known_ips.include?(ip)
+        if !excluded_ips.include?(ip) && !subnet.known_ips.include?(ip) && !is_ip_blocked?(ip)
           logger.debug("Found #{ip}")
+          block_ip(ip)
```

Comment:
Would you mind updating the debug line above saying "Found IP, blocking it for the next 30 minutes"?

Reply:
Done